pages/linux/apx-stacks.md

```diff
@@ -0,0 +1,29 @@
+# apx stacks
+
+> Manage stacks in `apx`.
+> Note: User created stack configurations are stored in `~/.local/share/apx/stacks`.
+> More information: <https://github.com/Vanilla-OS/apx>.
+
+- Interactively create a new stack configuration:
+
+`apx stacks new`
+
+- Interactively update a stack configuration:
+
+`apx stacks update {{name}}`
+
+- List all available stack configurations:
+
+`apx stacks list`
+
+- Remove a specified stack configuration:
+
+`apx stacks rm --name {{string}}`
+
+- Import a stack configuration:
+
+`apx stacks import --input {{path/to/stack.yml}}`
+
+- Export a stack configuration (Note: the output flag is optional):
```

Comment:
If the output flag is optional, what is the default? I think it should be added here.
```suggestion
- Export the stack configuration (Note: the output flag is optional):
```

Reply:
The file is outputted to your current directory by default, you can specify a different one using the output flag.